app/src/main/res/values/styles.xml

```diff
@@ -112,6 +112,8 @@
     </style>
 
     <style name="HedvigButton.Outlined.White">
+        <item name="android:paddingStart">12dp</item>
```

Comment:
I feel like this might be a little bit unexpected from the perspective of consuming this style: Would the user expect the padding to change by setting a different color on the button?